web/frontend/src/components/base.vue

```diff
@@ -454,7 +454,7 @@ onUnmounted(() => {
             :key="`base ${camera.name}`"
           >
             <Camera
-              v-show="openCameras[camera.name]"
+              v-if="openCameras[camera.name]"
```

Comment:
curious, even once the camera card shows, don't you still only want to enable once the toggle has been toggled?

Reply:
Apparently onMounted() is only called when v-if is true. Also since openCameras[camera.name] is tied to the value of the toggle switch this means the camera component will only mount when the toggle switch is toggled.

Reply:
gotcha